keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1059,16 +1052,20 @@ message NodePrepareResourceResponse {
   // These are the additional devices that kubelet must
   // make available via the container runtime. A resource
   // may have zero or more devices.
-  repeated CDIDevice devices = 1;
+  repeated string cdi_id = 1;
```

Comment:
What is the format of a single `cdi_id` string? Please add an explanation here.

Reply:
Currently it's something like <kind>=<name>, e.g. vendor1.com/gpu=gpu0, but CDI spec is not final and this can be changed. That's why I'm trying to use 'cdi_id' term here. Kubelet don't need to parse this id, it should just pass it down to CRI runtime.

Reply:
Then let's say "CDI id as defined by the CRI API".

Reply:
I'll use "Fully qualified CDI device name" as CDI devs call it here: https://github.com/container-orchestrated-devices/container-device-interface/blob/master/pkg/cdi/doc.go#L20-L24